utils/inspect-response.js

```diff
@@ -10,9 +10,38 @@ const prettyPrintResponse = (response, options) => {
     explorer.printTransactionUrl(txnId, options);
 };
 
+const prettyPrintError = (error, options) => {
+    console.log('An error occured');
+    console.log(formatResponse(error));
+    const txnId = getTxnIdFromError(error);
+    if (txnId) {
+        console.log(`We attempted to send transaction ${txnId} to NEAR, but something went wrong.`);
+        explorer.printTransactionUrl(txnId, options);
+        console.log('Note: if the transaction was invalid (e.g. not enough balance), it will show as Not started/Finalizing.');
```

Comment:
```suggestion
        console.log('Note: if the transaction was invalid (e.g. not enough balance), it will show as "Not started" or "Finalizing"');
```